### PR TITLE
Spanish language added

### PR DESCRIPTION
**Pull request by IsmaCortGtz**
# Spanish language added

I added the Spanish translation to the `README` file and all the `step-by-step` instructions.

The `spanish` badge was added to all `README` files.

The `How To Get` spanish button was added.

### CODE REVIEW

File: README.es.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+
+#### - Si necesitas más ayuda, haz clic en el botón `Cómo Obtener?` para obtener un tutorial paso a paso sobre cómo conseguir este logro.
+
+<div align="center"><a href="assets/steps-es/README.pull-shark-es.md"><img src="assets/img/btn-es.png" alt=""></a></div>
+<hr>
+
+<br>
+
+<div align="center"  >
+<a href="assets/steps/README.starstruck.md">
```
Comment (YasinDehfuli) [inline README.es.md:145]: You need to change href link to <a href="assets/steps-es/README.starstruck-es.md">

File: README.es.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+
+#### - Si necesitas más ayuda, haz clic en el botón `Cómo Obtener?` para obtener un tutorial paso a paso sobre cómo conseguir este logro.
+
+<div align="center"><a href="assets/steps-es/README.starstruck-es.md"><img src="assets/img/btn-es.png" alt=""></a></div>
+<hr>
+
+<br>
+
+<div align="center"  >
+<a href="assets/steps/README.pair-extraordinaire.md">
```
Comment (YasinDehfuli) [inline README.es.md:163]: you need to change href link to <a href="assets/steps-es/README.pair-extraordinaire-es.md">

File: README.es.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+
+#### - Si necesitas más ayuda, haz clic en el botón `Cómo Obtener?` para obtener un tutorial paso a paso sobre cómo conseguir este logro.
+
+<div align="center"><a href="assets/steps-es/README.galaxy-brain-es.md"><img src="assets/img/btn-es.png" alt=""></a></div>
+<hr>
+
+<br>
+
+<div align="center"  >
+<a href="assets/steps/README.pull-shark.md">
```
Comment (YasinDehfuli) [inline README.es.md:127]: change href link to <a href="assets/steps-es/README.pull-shark-es.md">

File: README.es.md [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+
+#### - Si necesitas más ayuda, haz clic en el botón `Cómo Obtener?` para obtener un tutorial paso a paso sobre cómo conseguir este logro.
+
+<div align="center"><a href="assets/steps-es/README.yolo-es.md"><img src="assets/img/btn-es.png" alt=""></a></div>
+<hr>
+
+<br>
+
+<div align="center"  >
+<a href="assets/steps/README.galaxy-brain.md">
```
Comment (YasinDehfuli) [inline README.es.md:109]: change href to <a href="assets/steps-es/README.galaxy-brain-es.md">

File: README.es.md [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+#### Aquí te mostramos cómo obtener los logros de GitHub paso a paso :)
+
+<br>
+
+# Lista de logros 📃
+
+<br>
+
+<div align="center"  >
+<a href="assets/steps/README.quickdraw.md">
```
Comment (YasinDehfuli) [inline README.es.md:73]: change href to <a href="assets/steps-es/README.quickdraw-es.md">

**Review by YasinDehfuli — COMMENTED** [review]
Please change image href links to button href

**Comment by YasinDehfuli** [comment]
Please check button href links and set image links to button links href

**Comment by IsmaCortGtz** [comment]
Damn, you're right. I'll update it. 😅

**Pull request merged**